### PR TITLE
Tweak: Editor V2 - Theme-Builder button [ED-9716]

### PR DESCRIPTION
**Pull request by rotemee**

### CODE REVIEW

File: core/editor/config-providers/editor-v2-config-provider.php [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Editor_V2_Config_Provider implements Config_Provider_Interface {
 		'elements-panel',
 		'structure',
 		'top-bar',
+		'theme-builder',
```
Comment (StyleShit) [inline core/editor/config-providers/editor-v2-config-provider.php:18]: a-z?

File: packages/packages/theme-builder/src/hooks/use-action-props.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { __ } from '@wordpress/i18n';
+import ThemeBuilderIcon from '../icons/theme-builder-icon';
+import { runCommand } from '@elementor/v1-adapters';
+
+export default function useDocumentPreviewProps() {
```
Comment (StyleShit) [inline packages/packages/theme-builder/src/hooks/use-action-props.tsx:5]: rename function and/or file name...

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: packages/packages/theme-builder/src/hooks/use-action-props.tsx (hunk 2 above)
Comment (rotemee) [reply packages/packages/theme-builder/src/hooks/use-action-props.tsx]: Done.

File: core/editor/config-providers/editor-v2-config-provider.php (hunk 1 above)
Comment (rotemee) [reply core/editor/config-providers/editor-v2-config-provider.php:17]: Done.

**Review by StyleShit — APPROVED** [review]

File: packages/packages/theme-builder/src/init.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+export default function init() {
+	registerTopBarMenuItems();
+}
+
+function registerTopBarMenuItems() {
+	registerAction( 'main', {
+		name: 'open-theme-builder',
+		useProps: useActionProps,
+	} );
```
Comment (Nevoss) [inline packages/packages/theme-builder/src/init.tsx:12]:
```suggestion
	registerAction( 'main', {
		name: 'open-theme-builder',
		props: {
			icon: ThemeBuilderIcon,
			title: __( 'Theme Builder', 'elementor' ),
			onClick: () => runCommand( 'app/open' ),
		}
	} );
```

**Review by Nevoss — APPROVED** [review]

File: packages/packages/theme-builder/src/init.tsx (hunk 3 above)
Comment (rotemee) [reply packages/packages/theme-builder/src/init.tsx:12]: @Nevoss - For now I'll keep it as it is for the unit testing until we'll decide on a way for testing it.

**Pull request merged**